public/styles/top.css

```diff
@@ -38,7 +38,7 @@ border-radius: always 4px
 
 */
 
-@import url(https://fonts.googleapis.com/css?family=Roboto+Slab:300,400,700);
+@import url(https://fonts.googleapis.com/css?family=Roboto+Slab:400,700);
```

Comment:
Our idea actually was that we don't modify the top.css, so that it is easier to keep in sync with the one from dlang.org